npm/iptm/ioshim_fake.go

```diff
@@ -0,0 +1,77 @@
+package iptm
+
+import (
+	"io"
+	"io/fs"
+	"testing/fstest"
+
+	testingutils "github.com/Azure/azure-container-networking/test/utils"
+)
+
+type fakeIptOperationShim struct {
+	configname string
+	m          fstest.MapFS
```

Comment:
I am wondering whether m and fo are necessary? Seems we can take the same approaches (e.g., just returning nil or etc). Are there any reason to have them?

Reply:
they are used to create an in memory file structure for tests to behave as closest as a real fs as possible